### DISCUSSION TITLE
plotting Stochastic

### DISCUSSION THREAD

**Discussion by sultank1** [kernc/backtesting.py#701] [Q&A]
hi ,,
is there way to plot the stochastic K and D values in same separate section right now both K and D graph are plotted separate...

i have read the documentation but did not found the answer..

**Comment by kernc** [comment]
Should also search the [FAQ sections](https://kernc.github.io/backtesting.py/doc/backtesting/index.html#faq). See https://github.com/kernc/backtesting.py/issues/88.

**Comment by sultank1** [comment]
Thanks i will
